cluster/manifests/admission-control/teapot.yaml

```
@@ -14,7 +14,7 @@ webhooks:
       - operations: [ "CREATE" ]
         apiGroups: [""]
         apiVersions: ["v1"]
-        resources: ["pods"]
+        resources: ["pods", "pods/binding"]
```

Please split it into a separate rule, with `failurePolicy: Ignore`.

---

good idea - does this mean we could propagate the errors from binding (again)?
Might make troubleshooting easier.

---

anyways, I've changed as advised.